senator/game.py

```diff
@@ -102,21 +109,29 @@ def solve_node_via_rm_plus(self, node: Node, iterations: int = 1000) -> None:
         if (conv := node.nash_conv()) >= 0.001:
             print(f"Nash Convexity is {conv}, something is wrong...")
 
-    def run_greedy_search(self, num_expansions: int = 10):
-        """
-        Run a greedy search algorithm, periodically expanding the tree. Each expansion is done by first solving the
-        node using the RM+ algorithm, sampling a joing action, and adding the child to the tree.
-        TODO: decide how to do the expanding. Baseline is to just do a path of length 5.
-        """
```

Comment:
Tomuhle se říká docstring, a měl by dokumentovat co funkce dělá, co vrací, co znamenají jaké její parametry atd. Neříkám že by to muselo být u každé funkce, ale většinou je dobrej nápad ho psát. Až to budeš číst za týden tak se ti to bude hodit :D

Reply:
PS docstring můžeš napsat i k třídě, třeba 
```python
class Foo:
    """This is class Foo, it does foo stuff."""
```

Reply:
Průběžně doplním.